src/util/sandbox/fd_sandbox.c

```diff
@@ -40,18 +40,29 @@
 
 #endif
 
+void
+fd_sandbox_private_switch_uid_gid( uint desired_uid,
+                                   uint desired_gid );
+
 static int
-check_unshare_eacces_main( void * arg ) {
-  (void)arg;
+check_unshare_eacces_main( void * _arg ) {
+  ulong arg = (ulong)_arg;
+  uint desired_uid = (uint)((arg >>  0UL) & 0xFFFFUL);
+  uint desired_gid = (uint)((arg >> 32UL) & 0xFFFFUL);
 
+  fd_sandbox_private_switch_uid_gid( desired_uid, desired_gid );
   int result = unshare( CLONE_NEWUSER );
-  if( -1==result && errno==EACCES ) return 999;
+  if( -1==result && errno==EACCES ) return 255;
```

Comment:
Why not just return and check for EACCES instead of an arbitrary number? That at least communicates the intent.